db/upgrade_migrations/20160723012349_upgrade_v0_6_to_v0_7.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class UpgradeV06ToV07 < ActiveRecord::Migration
+  def up
+    Thredded::MessageboardGroup.transaction do
+      Thredded::MessageboardGroup.where(
+        name: Thredded::MessageboardGroup.group(:name).having('count(id) > 1').select(:name)
+      ).group_by(&:name).each_value do |messageboard_groups|
+        messageboard_groups.each_with_index do |messageboard_group, i|
+          messageboard_group.update!(name: "#{messageboard_group.name}-#{i + 2}")
+        end
+      end
+    end
```

Comment:
If you want the suffix to look like this, simply replace this:

```ruby
messageboard_groups.each_with_index do |messageboard_group, i|
```

with

```ruby
messageboard_groups.from(1).each_with_index do |messageboard_group, i|
```

and `i + 2` -> `i + 1`

in the migration